### PR TITLE
Seqr proportional map billing dashboard + revised endpoints

### PR DESCRIPTION
**Pull request by illusional**
- Move the propmap function into metamist (and rewrite for sequencing group support)
- Provide a mechanism for visualising the result.
- New endpoints to expose for cpg-infra change

### CODE REVIEW

**Comment by illusional** [comment]
@danielreti, could you please review front-end changes
@violetbrina, could you review anything Python related please

File: web/src/pages/billing/SeqrProportionalMapGraph.tsx [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+        return <Message negative>No data found</Message>
+    }
+
+    const selectedProjects: string[] = projectSelections
+        ? _.sortBy(Object.keys(projectSelections).filter((project) => projectSelections[project]))
+        : []
+
+    let graphComponent: React.ReactElement | undefined = undefined
+
+    if (allPropMapData) {
```
Comment (danielreti) [inline web/src/pages/billing/SeqrProportionalMapGraph.tsx:168]: It's probably neater to put a catch in the StackedAreaChart to return an empty fragment if there is no data

**Review by danielreti — APPROVED** [review]
Approved with a small change

File: web/src/pages/billing/SeqrProportionalMapGraph.tsx (hunk 1 above)
Comment (illusional) [reply web/src/pages/billing/SeqrProportionalMapGraph.tsx:168]: Yeah good call, actioned :)

Comment (illusional) [reply web/src/pages/billing/SeqrProportionalMapGraph.tsx]: Good shout, have actioned this.

**Review by violetbrina — APPROVED** [review]

**Pull request merged**